### PR TITLE
feat: Merged contact module with profile

### LINKED ISSUE #3142: Merge profile with "Contact info" (Settings)
**Describe the feature**
Related parent issue https://github.com/fossasia/open-event-frontend/issues/3114

**Expected behaviour**
Contact info should be under profile, rather than having a separate tab

**Additional context**
Working on this.

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Screenshot 2019-06-18 at 3 44 18 AM
Fixes #3142, #3114 

#### Short description of what this resolves:
 Merges contact info with profile module

#### Changes proposed in this pull request:

- Shifted fields to profile component
- Modified tests accordingly
- Moved validation rules
- Made the spacing & placement similar to other tabs
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/account/profile.js [hunk 1]
```diff
@@ -1,4 +1,20 @@
 import Controller from '@ember/controller';
 
 export default Controller.extend({
+  actions: {
+    updateContactInfo() {
+      this.set('isLoading', true);
+      let currentUser = this.model;
```
Comment (niranjan94) [inline app/controllers/account/profile.js]: Why do you need to store it to a separate variable ? You can directly use it

File: app/controllers/account/profile.js [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+      currentUser.save()
```
Comment (niranjan94) [inline app/controllers/account/profile.js]: IMO async-await would make this less verbose

Comment (mrsaicharan1) [reply app/controllers/account/profile.js]: @niranjan94 I agree. Made it easier like that.

File: app/controllers/account/profile.js [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
 export default Controller.extend({
+  actions: {
+    async updateContactInfo() {
+      this.set('isLoading', true);
+      try {
+        await this.model.save();
+        this.notify.success(this.l10n.t('Your Contact Info has been updated'));
+      }  catch (error) {
+        this.notify.error(this.l10n.t(error.message));
+      } finally {
```
Comment (niranjan94) [inline app/controllers/account/profile.js]: Finally is not needed here at all. `this.set('isLoading', false);` can be out of the try-catch block and it'll be executed anyway either on success or fail.

Comment (mrsaicharan1) [reply app/controllers/account/profile.js]: @niranjan94 Updated. Please have a look

**Comment by mrsaicharan1** [comment]
@uds5501 @shreyanshdwivedi @CosmicCoder96 @niranjan94 Updated. Please have a look

**Comment by mariobehling** [comment]
Please make the following changes as well:
* Adjust to the width style of the other tabs (full width of area). We can later decide if we want to change all of them, but for now keep them in the same way, please.
* Make phone number optional - not required. Therefore take out the *.

Screenshot from 2019-06-17 14-19-53

**Comment by mrsaicharan1** [comment]
@mariobehling 
Screenshot 2019-06-11 at 4 00 18 PM
Screenshot 2019-06-14 at 4 53 40 PM

Here's the updated screenshot. Edited it in the description as well

**Comment by mrsaicharan1** [comment]
Screenshot 2019-06-17 at 11 41 51 PM
Screenshot 2019-06-17 at 11 42 40 PM
Here are the updated screenshots

**Comment by mariobehling** [comment]
Keep the photo on the left, but please use the whole area.

**Comment by mariobehling** [comment]
Just use the full width.

**Comment by mrsaicharan1** [comment]
Screenshot 2019-06-18 at 3 44 18 AM

**Comment by mrsaicharan1** [comment]
Screenshot 2019-06-18 at 3 44 18 AM
 
Made it uniform with the other tabs. Please have a look @CosmicCoder96

**Pull request merged**